server/channels/app/platform/shared_channel_notifier.go

```diff
@@ -58,7 +58,7 @@ func (ps *PlatformService) SharedChannelSyncHandler(event *model.WebSocketEvent)
 func isEligibleForEvents(syncService SharedChannelServiceIFace, event *model.WebSocketEvent, events model.StringArray) bool {
```

Comment:
I'm still wondering if we can change the signature of `isEligibleForEvents` to allow a `[]model.WebsocketEventType`, but that is not a blocking concern.